demo/WebexMeetingsWidgetDemo.jsx

```diff
@@ -109,6 +125,19 @@ export default function WebexMeetingsWidgetDemo({token, fedramp}) {
   );
 }
 
+const getHydraId = (destination) => {
+  const {type, id, cluster} = deconstructHydraId(destination);
+  const UUID_REG = /^[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}$/i;
+
+  if (id && UUID_REG.test(id)) {
+    if (type === "ROOM") {
+      return {room: true, destination: id, cluster};
+    }
+    return {};
```

Comment:
This could be removed, it's redundant.

Reply:
Could be simplified as below

```
if (id && UUID_REG.test(id) && type === "ROOM") {
  return {room: true, destination: id, cluster};
}

return {};
```

Reply:
Correct @rarajes2, will update. PR was in draft, will do the cleanup. we took the method from js SDK and used it directly.

Reply:
@rarajes2, I have pushed the changes with modifications, Also added comments.